platform/ui/src/components/downloadDialog/DownloadDialog.js

```diff
@@ -212,18 +224,22 @@ const DownloadDialog = ({
               <TextInput
                 type="number"
                 min={minimumSize}
+                max={maximumSize}
                 value={width}
                 label={t('Image width (px)')}
                 onChange={onWidthChange}
+                onBlur={onHeightChange}
               />
             </div>
             <div className="height">
               <TextInput
                 type="number"
                 min={minimumSize}
+                max={maximumSize}
                 value={height}
                 label={t('Image height (px)')}
                 onChange={onHeightChange}
+                onBlur={onHeightChange}
```

Comment:
Does `onBlur` catch anything not already caught by `onChange`?

Reply:
It wasn't, I forgot to take this out. Thank you.